custom-recipes/googlesheets-append/recipe.json

```diff
@@ -27,12 +27,34 @@
     ],
 
     "params": [
+        {
+            "name": "auth_type",
+            "label": "Type of authentication",
+            "type": "SELECT",
+            "selectChoices": [
+                {
+                    "value": "service-account",
+                    "label": "Service account preset"
+                },
+                {
+                    "value": "credentials",
+                    "label": "Service account credentials (legacy)"
+                }
+            ]
+        },
```

Comment:
I kind of think, either both of these choices should have keys that match the parameters' names, or have slightly more specific/explanatory names like "legacy-service-account" and "preset-service-account"
At the moment both "service-account", and "credential" seem ambiguous (I know you can't change "credential" in the existing param but you can here, right?)

Reply:
how about `service-account-preset` and `service-account-inline` ?

Reply:
Changed to `preset-service-account` and `legacy-service-account` in [0ae3fbb](https://github.com/dataiku/dss-plugin-googlesheets/pull/2/commits/0ae3fbb57d6aee92b5fdfdcf37504229875a8a26)